docs/contributors/native-mobile.md

```diff
@@ -0,0 +1,6 @@
+# React Native based mobile Gutenberg
+Interwined with the web codepaths, the Gutenberg repo also includes the React Native based mobile tree. The mobile client is packaged and released via the [official WordPress apps](https://wordpress.org/mobile/). Even though the build pipeline is rather different and lives in its own repo for now ([here's the native mobile repo](https://github.com/wordpress-mobile/gutenberg-mobile)), the source code itself is taken directly from this repo and the "web" side codepaths.
+## Native mobile specific files
+The majority of the code shared with native mobile is in the very same JavaScript module and SASS style files. In the cases where the code paths need to diverge, a `.native.js` or `.native.scss` variant of the file is created. In some cases, platform specific files can be also found for Android (`.android.js`) or iOS (`.ios.js`).
```

Comment:
```suggestion
The majority of the code shared with native mobile is in the very same JavaScript module and SASS style files. In the cases where the code paths need to diverge, a `.native.js` or `.native.scss` variant of the file is created. In some cases, platform-specific files can be also found for Android (`.android.js`) or iOS (`.ios.js`).
```

Reply:
Negatives are hard 😸 ! Thanks for spotting @gziolo !